### PR TITLE
Use alternative selector script

### PR DESCRIPTION
**Pull request by gosha212**
## Description

- This pull request addresses the issue described here: #<?>

In this pull request, I have …

---

> _For features/enhancements:_
 - [ ] I have added/updated the relevant references in the [documentation](https://github.com/wix/Detox/tree/master/docs) files.

> _For API changes:_
 - [ ] I have made the necessary changes in the [types index](https://github.com/wix/Detox/blob/master/detox/index.d.ts) file.

### CODE REVIEW

File: detox/android/detox/src/full/java/com/wix/detox/espresso/web/DetoxDriverAtoms.java [hunk 1]
```diff
@@ -0,0 +1,642 @@
...
+     * Returns {@code true} if the desired element is in view after scrolling.
+     */
+    public static Atom<Boolean> webScrollIntoView() {
+        return new WebScrollIntoViewAtom(new WebScrollIntoViewSimpleAtom(), castOrDie(Boolean.class));
+    }
+
+    /**
+     * Finds multiple elements given a locator strategy.
+     */
+    public static Atom<List<ElementReference>> findMultipleElementsDetox(
```
Comment (asafkorem) [inline detox/android/detox/src/full/java/com/wix/detox/espresso/web/DetoxDriverAtoms.java:151]: except for this method I see lots of methods here that we're not using, copied from the original that we're still using. Can we keep only the specific custom method we're using?

**Review by asafkorem — CHANGES_REQUESTED** [review]
@gosha212 is that the solution we're moving forward or we're still testing this approach?
Anyhow if it works on Android, and the same script from iOS, let's plan together how we can do that from the same source (JS level)

File: detox/android/detox/src/full/java/com/wix/detox/espresso/web/DetoxDriverAtoms.java (hunk 1 above)
Comment (gosha212) [reply detox/android/detox/src/full/java/com/wix/detox/espresso/web/DetoxDriverAtoms.java:151]: This is still WIP @asafkorem

**Comment by gosha212** [comment]
This POC is not relevant. Couldn't make it work better that the espresso solution

**Pull request closed without merging**